app/src/org/commcare/android/util/ACRAUtil.java

```diff
@@ -0,0 +1,21 @@
+package org.commcare.android.util;
+
+import org.acra.ACRA;
+import org.acra.ErrorReporter;
+
+/**
+ * Created by wpride1 on 3/3/15.
```

Comment:
Replace comment template with a real comment, maybe something like:
/** 
* Specifies the custom data collected by ACRA error reporting.
*
* @author wpride
*/